docs/table/mixin_columns.rst

```diff
@@ -211,12 +211,13 @@ that contain mixin columns:
    * - :ref:`grouped-operations`
      - Not implemented yet, but no fundamental limitation
    * - :ref:`stack-vertically`
-     - Not implemented yet, pending definition of generic concatenation protocol
+     - Available for `~astropy.units.Quantity` and any other mixin classes that provide an
+       `empty_like() method`_ in the ``info`` descriptor.
```

Comment:
Either here or below (probably better below), it should also be noted that the class has to allow setting elements (though I guess it is somewhat obvious...)